styles.css

```diff
@@ -0,0 +1,184 @@
+/* ==============
+   UTILITY STYLES
+   ============== */
+
+/* Removes the default bullets and spacing from a list */
+.u-list-reset {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+}
+
+/* Displays list items as inline block  */
+.u-list-inline li {
+    display: inline-block;
+}
+
+/* width */
+::-webkit-scrollbar {
+    width: 2px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+    background: lightgrey;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+    background: gray;
+    width: 2px;
+}
+
+/* ==============
+   BASE STYLES
+   ============== */
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    /* background-color: greenyellow; */
+    color: hsl(190, 5%, 15%);
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
+        Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    /* system font stack */
+    /* page seetings / frames */
+    /* padding: 2em 2em 2em 2em; */
+    text-align: left;
```

Comment:
This reiterates the  [default browser styles](https://browserdefaultstyles.com/) for body and, therefore, can be removed:
```suggestion
```